cmd/issuer-rest/static/preAuthorize.html

```diff
@@ -33,43 +64,48 @@
                 {{end}}
             </select>
         </div>
-    </div>
-    <div class="w-full p-4">
-        <div class="grid place-items-center">
+        <div class="w-full p-4 grid place-items-center">
             <label id="txState">State: awaiting QR code scan</label>
         </div>
-    </div>
-    <div class="w-full p-4">
-        <div class="grid place-items-center">
+        <div class="w-full p-4 grid place-items-center">
             <div id="qrCode"></div>
         </div>
-    </div>
-    <div class="w-full p-4" id="pinBox">
-        <div class="grid place-items-center">
-            <div>
-                <label for="pin2" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">PIN</label>
-                <input type="text" id="pin2"
-                       class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                       placeholder="{{.Pin}}" readonly>
+        <div class="w-full p-4" id="pinBox">
+            <div class="grid place-items-center">
+                <div>
+                    <label for="pin2" class="block mb-2 text-lg font-semibold text-gray-900">PIN</label>
+                    <button class="flex flex-row items-center" onclick="copyToClipboard('pin2')">
+                        <input type="text" id="pin2"
+                               class="cursor-pointer bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg text-center w-full p-2.5"
+                               value="{{.Pin}}" readonly disabled>
```

Comment:
Can we add PIN text and PIN field value in same line ? Currently, it looks bit odd with them being in different lines.

Reply:
Or maybe centered above the field?